includes/alerts/transport.canopsis.php

```diff
@@ -0,0 +1,64 @@
+require_once "/usr/share/php/PhpAmqpLib/autoload.php";
```

Comment:
Hum.... Good question. As far as I can see, only Ubuntu and Debian bring amqplib-php. I don't find it on RHEL like distribs...There is no reason that Ubuntu package change this location... Xenial package will keep the same path.

If people install amqplib-php on distribs other than Debian like, by compiling it for example, it will certainly not be in the same path ... I thought notify users of this dependency during install process but install process of libreNMS is just a git clone, so not possible to hot compile an amqplib-php during install of libreNMS.

What do you think ?